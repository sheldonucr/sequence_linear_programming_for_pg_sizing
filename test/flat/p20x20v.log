
******** PCx version 1.1 (Nov 1997) ************

Problem 'from_lp_fi' terminated with OPTIMAL status
Iterations=21, Termination Code=0

MPS formulation has 878 rows, 401 columns

PARAMETER SUMMARY
=================

Maximum number of iterations: 200
Tolerances: Opt=1.00e-10  PriFeas=1.00e-08 DualFeas=1.00e-08
Gondzio strategy selected:  Maximum Gondzio corrections = 0
NO iterative refinement
Presolving was performed:
   Before Presolving:  878 rows, 1279 columns
   After  Presolving:  878 rows, 1278 columns  (1 pass)
MINIMIZE the objective
Solution written to output file p20x20v.out


FACTORIZATION SUMMARY
=====================

code used: Ng-Peyton sparse Cholesky library
Nonzeros in L=3293;  Density of L=0.007404

ITERATION SUMMARY
=================

 Iter    Primal       Dual      (PriInf  DualInf)  log(mu)   Merit
  0   8.3944e+06   2.6927e+06  (9.8e-01 5.5e-08)     7.74    2.2e+01
  1  -1.4967e+08  -5.1944e+08  (4.4e-01 2.7e-08)     7.42    1.2e+01
  2   4.7913e+07  -7.6637e+08  (2.0e-01 1.8e-08)     7.18    9.1e+00
  3   4.5950e+06  -6.6767e+08  (4.2e-02 4.4e-09)     6.56    3.8e+00
  4   6.5184e+06  -1.7622e+08  (3.5e-03 1.2e-10)     5.48    6.5e-01
  5   6.8163e+06  -3.6356e+07  (7.8e-05 6.8e-12)     4.57    1.4e-01
  6   6.5074e+06  -2.1287e+06  (3.3e-13 1.3e-12)     3.85    2.9e-02
  7   6.5063e+06   3.1581e+06  (3.0e-12 6.7e-22)     3.42    1.1e-02
  8   5.9811e+06   5.2930e+06  (4.1e-10 3.2e-22)     2.73    2.2e-03
  9   5.8933e+06   5.6846e+06  (3.7e-10 5.5e-23)     2.21    6.8e-04
 10   5.8858e+06   5.7518e+06  (1.3e-10 4.8e-23)     2.02    4.4e-04
 11   5.8788e+06   5.8248e+06  (2.1e-10 1.1e-23)     1.63    1.8e-04
 12   5.8708e+06   5.8579e+06  (3.0e-10 4.2e-24)     1.00    4.2e-05
 13   5.8702e+06   5.8661e+06  (4.9e-10 1.7e-24)     0.50    1.3e-05
 14   5.8700e+06   5.8682e+06  (2.5e-10 5.1e-24)     0.13    5.6e-06
 15   5.8698e+06   5.8689e+06  (2.1e-10 2.6e-24)    -0.17    2.8e-06
 16   5.8697e+06   5.8694e+06  (1.2e-10 1.0e-24)    -0.55    1.2e-06
 17   5.8697e+06   5.8695e+06  (9.5e-11 2.5e-24)    -0.90    5.2e-07
 18   5.8696e+06   5.8696e+06  (1.2e-10 2.5e-24)    -1.28    2.2e-07
 19   5.8696e+06   5.8696e+06  (1.4e-10 3.3e-25)    -1.93    5.3e-08
 20   5.8696e+06   5.8696e+06  (1.4e-10 2.7e-24)    -2.95    5.1e-09
 21   5.8696e+06   5.8696e+06  (5.9e-11 1.1e-24)    -4.89    9.0e-10

 21 iterations

Terminated with status OPTIMAL (code 0)

Primal Objective = -1.61172796e+06
Dual   Objective = -1.61172815e+06

Complementarity          =  1.66e-02
Relative Complementarity =  2.83e-09

Relative Infeasibilities:
Primal = 6.139e-13,    Dual   = 0.000e+00.

TIME SUMMARY
============

Time to read input file: 0.14 sec
Time to presolve       : 0.04 sec

Time to solve          : 0.66 sec

