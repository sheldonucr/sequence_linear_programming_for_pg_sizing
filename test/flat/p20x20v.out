Solution for 'from_lp_fi'
 V_1        4.9974258e+00  
 V_2        4.9972564e+00  
 V_3        4.9971098e+00  
 V_4        4.9969787e+00  
 V_5        4.9968634e+00  
 V_6        4.9967695e+00  
 V_7        4.9966943e+00  
 V_8        4.9966001e+00  
 V_9        4.9965150e+00  
 V_100      4.9984152e+00  
 V_101      4.9974281e+00  
 V_102      4.9972626e+00  
 V_103      4.9971100e+00  
 V_104      4.9969860e+00  
 V_105      4.9968739e+00  
 V_106      4.9967805e+00  
 V_107      4.9967097e+00  
 V_108      4.9966168e+00  
 V_109      4.9965330e+00  
 V_110      4.9964882e+00  
 V_111      4.9964896e+00  
 V_112      4.9965330e+00  
 V_113      4.9966162e+00  
 V_114      4.9967432e+00  
 V_115      4.9969145e+00  
 V_116      4.9971324e+00  
 V_117      4.9973931e+00  
 V_118      4.9977007e+00  
 V_119      4.9980346e+00  
 V_120      4.9983870e+00  
 V_121      4.9974217e+00  
 V_122      4.9972586e+00  
 V_123      4.9971089e+00  
 V_124      4.9969798e+00  
 V_125      4.9968695e+00  
 V_126      4.9967783e+00  
 V_127      4.9967013e+00  
 V_128      4.9966190e+00  
 V_129      4.9965353e+00  
 V_130      4.9964899e+00  
 V_131      4.9964951e+00  
 V_132      4.9965397e+00  
 V_133      4.9966328e+00  
 V_134      4.9967703e+00  
 V_135      4.9969398e+00  
 V_136      4.9971579e+00  
 V_137      4.9974036e+00  
 V_138      4.9977093e+00  
 V_139      4.9980381e+00  
 V_140      4.9983971e+00  
 V_141      4.9973790e+00  
 V_142      4.9972141e+00  
 V_143      4.9970696e+00  
 V_144      4.9969374e+00  
 V_145      4.9968205e+00  
 V_146      4.9967241e+00  
 V_147      4.9966507e+00  
 V_148      4.9965965e+00  
 V_149      4.9965134e+00  
 V_150      4.9964717e+00  
 V_151      4.9964690e+00  
 V_152      4.9965090e+00  
 V_153      4.9965919e+00  
 V_154      4.9967168e+00  
 V_155      4.9968873e+00  
 V_156      4.9970994e+00  
 V_157      4.9973597e+00  
 V_158      4.9976748e+00  
 V_159      4.9980330e+00  
 V_160      4.9983931e+00  
 V_161      4.9974225e+00  
 V_162      4.9972580e+00  
 V_163      4.9971040e+00  
 V_164      4.9969779e+00  
 V_165      4.9968656e+00  
 V_166      4.9967690e+00  
 V_167      4.9966936e+00  
 V_168      4.9965988e+00  
 V_169      4.9965115e+00  
 V_170      4.9964688e+00  
 V_171      4.9964679e+00  
 V_172      4.9965116e+00  
 V_173      4.9965976e+00  
 V_174      4.9967289e+00  
 V_175      4.9969097e+00  
 V_176      4.9971248e+00  
 V_177      4.9973778e+00  
 V_178      4.9976862e+00  
 V_179      4.9980230e+00  
 V_180      4.9983642e+00  
 V_181      4.9973985e+00  
 V_182      4.9972343e+00  
 V_183      4.9970793e+00  
 V_184      4.9969511e+00  
 V_185      4.9968394e+00  
 V_186      4.9967467e+00  
 V_187      4.9966734e+00  
 V_188      4.9966076e+00  
 V_189      4.9965211e+00  
 V_190      4.9964734e+00  
 V_191      4.9964720e+00  
 V_192      4.9965134e+00  
 V_193      4.9966009e+00  
 V_194      4.9967300e+00  
 V_195      4.9968969e+00  
 V_196      4.9971236e+00  
 V_197      4.9973765e+00  
 V_198      4.9976781e+00  
 V_199      4.9980175e+00  
 V_200      4.9983912e+00  
 V_201      4.9973804e+00  
 V_202      4.9972117e+00  
 V_203      4.9970659e+00  
 V_204      4.9969299e+00  
 V_205      4.9968206e+00  
 V_206      4.9967280e+00  
 V_207      4.9966558e+00  
 V_208      4.9966019e+00  
 V_209      4.9965199e+00  
 V_210      4.9964747e+00  
 V_211      4.9964721e+00  
 V_212      4.9965136e+00  
 V_213      4.9965980e+00  
 V_214      4.9967334e+00  
 V_215      4.9969075e+00  
 V_216      4.9971315e+00  
 V_217      4.9973970e+00  
 V_218      4.9976937e+00  
 V_219      4.9980440e+00  
 V_220      4.9983814e+00  
 V_221      4.9973977e+00  
 V_222      4.9972297e+00  
 V_223      4.9970853e+00  
 V_224      4.9969555e+00  
 V_225      4.9968433e+00  
 V_226      4.9967504e+00  
 V_227      4.9966737e+00  
 V_228      4.9966065e+00  
 V_229      4.9965168e+00  
 V_230      4.9964715e+00  
 V_231      4.9964682e+00  
 V_232      4.9965099e+00  
 V_233      4.9965912e+00  
 V_234      4.9967251e+00  
 V_235      4.9968977e+00  
 V_236      4.9971102e+00  
 V_237      4.9973585e+00  
 V_238      4.9976496e+00  
 V_239      4.9980105e+00  
 V_240      4.9983822e+00  
 V_241      4.9973785e+00  
 V_242      4.9972090e+00  
 V_243      4.9970659e+00  
 V_244      4.9969336e+00  
 V_245      4.9968195e+00  
 V_246      4.9967309e+00  
 V_247      4.9966564e+00  
 V_248      4.9966014e+00  
 V_249      4.9965263e+00  
 V_250      4.9964799e+00  
 V_251      4.9964785e+00  
 V_252      4.9965219e+00  
 V_253      4.9966061e+00  
 V_254      4.9967331e+00  
 V_255      4.9969106e+00  
 V_256      4.9971208e+00  
 V_257      4.9973844e+00  
 V_258      4.9976793e+00  
 V_259      4.9980154e+00  
 V_260      4.9983959e+00  
 V_261      4.9974035e+00  
 V_262      4.9972384e+00  
 V_263      4.9970894e+00  
 V_264      4.9969597e+00  
 V_265      4.9968523e+00  
 V_266      4.9967556e+00  
 V_267      4.9966829e+00  
 V_268      4.9966144e+00  
 V_269      4.9965275e+00  
 V_270      4.9964833e+00  
 V_271      4.9964812e+00  
 V_272      4.9965223e+00  
 V_273      4.9966103e+00  
 V_274      4.9967419e+00  
 V_275      4.9969076e+00  
 V_276      4.9971149e+00  
 V_277      4.9973804e+00  
 V_278      4.9976933e+00  
 V_279      4.9980360e+00  
 V_280      4.9983801e+00  
 V_281      4.9973936e+00  
 V_282      4.9972310e+00  
 V_283      4.9970843e+00  
 V_284      4.9969555e+00  
 V_285      4.9968407e+00  
 V_286      4.9967457e+00  
 V_287      4.9966734e+00  
 V_288      4.9966161e+00  
 V_289      4.9965264e+00  
 V_290      4.9964830e+00  
 V_291      4.9964805e+00  
 V_292      4.9965240e+00  
 V_293      4.9966091e+00  
 V_294      4.9967411e+00  
 V_295      4.9969060e+00  
 V_296      4.9971241e+00  
 V_297      4.9973738e+00  
 V_298      4.9976845e+00  
 V_299      4.9980173e+00  
 V_300      4.9983802e+00  
 V_301      4.9974109e+00  
 V_302      4.9972479e+00  
 V_303      4.9970971e+00  
 V_304      4.9969681e+00  
 V_305      4.9968569e+00  
 V_306      4.9967626e+00  
 V_307      4.9966908e+00  
 V_308      4.9966150e+00  
 V_309      4.9965262e+00  
 V_310      4.9964806e+00  
 V_311      4.9964784e+00  
 V_312      4.9965198e+00  
 V_313      4.9966040e+00  
 V_314      4.9967338e+00  
 V_315      4.9969090e+00  
 V_316      4.9971216e+00  
 V_317      4.9973844e+00  
 V_318      4.9976821e+00  
 V_319      4.9980335e+00  
 V_320      4.9983728e+00  
 V_321      4.9974226e+00  
 V_322      4.9972531e+00  
 V_323      4.9971090e+00  
 V_324      4.9969751e+00  
 V_325      4.9968688e+00  
 V_326      4.9967754e+00  
 V_327      4.9967030e+00  
 V_328      4.9966173e+00  
 V_329      4.9965254e+00  
 V_330      4.9964810e+00  
 V_331      4.9964794e+00  
 V_332      4.9965221e+00  
 V_333      4.9966045e+00  
 V_334      4.9967280e+00  
 V_335      4.9969042e+00  
 V_336      4.9971174e+00  
 V_337      4.9973796e+00  
 V_338      4.9976853e+00  
 V_339      4.9980233e+00  
 V_340      4.9984075e+00  
 V_341      4.9974017e+00  
 V_342      4.9972307e+00  
 V_343      4.9970873e+00  
 V_344      4.9969612e+00  
 V_345      4.9968503e+00  
 V_346      4.9967567e+00  
 V_347      4.9966838e+00  
 V_348      4.9966213e+00  
 V_349      4.9965335e+00  
 V_350      4.9964876e+00  
 V_351      4.9964897e+00  
 V_352      4.9965341e+00  
 V_353      4.9966197e+00  
 V_354      4.9967447e+00  
 V_355      4.9969119e+00  
 V_356      4.9971271e+00  
 V_357      4.9973938e+00  
 V_358      4.9976865e+00  
 V_359      4.9980428e+00  
 V_360      4.9983999e+00  
 V_361      4.9974217e+00  
 V_362      4.9972521e+00  
 V_363      4.9971027e+00  
 V_364      4.9969751e+00  
 V_365      4.9968652e+00  
 V_366      4.9967731e+00  
 V_367      4.9967021e+00  
 V_368      4.9966213e+00  
 V_369      4.9965343e+00  
 V_370      4.9964876e+00  
 V_371      4.9964859e+00  
 V_372      4.9965277e+00  
 V_373      4.9966132e+00  
 V_374      4.9967394e+00  
 V_375      4.9969113e+00  
 V_376      4.9971193e+00  
 V_377      4.9973839e+00  
 V_378      4.9976763e+00  
 V_379      4.9980190e+00  
 V_380      4.9983849e+00  
 V_381      4.9974141e+00  
 V_382      4.9972508e+00  
 V_383      4.9971011e+00  
 V_384      4.9969762e+00  
 V_385      4.9968663e+00  
 V_386      4.9967699e+00  
 V_387      4.9966930e+00  
 V_388      4.9966221e+00  
 V_389      4.9965297e+00  
 V_390      4.9964846e+00  
 V_391      4.9964837e+00  
 V_392      4.9965275e+00  
 V_393      4.9966106e+00  
 V_394      4.9967424e+00  
 V_395      4.9969174e+00  
 V_396      4.9971211e+00  
 V_397      4.9973746e+00  
 V_398      4.9976785e+00  
 V_399      4.9980284e+00  
 V_400      4.9984220e+00  
 V_401      5.0000000e+00  
 V_10       4.9964744e+00  
 V_11       4.9964750e+00  
 V_12       4.9965210e+00  
 V_13       4.9966059e+00  
 V_14       4.9967374e+00  
 V_15       4.9969122e+00  
 V_16       4.9971314e+00  
 V_17       4.9973918e+00  
 V_18       4.9976867e+00  
 V_19       4.9980388e+00  
 V_20       4.9983991e+00  
 V_21       4.9974092e+00  
 V_22       4.9972404e+00  
 V_23       4.9970881e+00  
 V_24       4.9969611e+00  
 V_25       4.9968494e+00  
 V_26       4.9967591e+00  
 V_27       4.9966857e+00  
 V_28       4.9966171e+00  
 V_29       4.9965305e+00  
 V_30       4.9964834e+00  
 V_31       4.9964804e+00  
 V_32       4.9965220e+00  
 V_33       4.9966073e+00  
 V_34       4.9967349e+00  
 V_35       4.9969119e+00  
 V_36       4.9971313e+00  
 V_37       4.9973944e+00  
 V_38       4.9976845e+00  
 V_39       4.9980374e+00  
 V_40       4.9983842e+00  
 V_41       4.9974249e+00  
 V_42       4.9972598e+00  
 V_43       4.9971139e+00  
 V_44       4.9969884e+00  
 V_45       4.9968765e+00  
 V_46       4.9967797e+00  
 V_47       4.9967071e+00  
 V_48       4.9966280e+00  
 V_49       4.9965362e+00  
 V_50       4.9964901e+00  
 V_51       4.9964885e+00  
 V_52       4.9965318e+00  
 V_53       4.9966150e+00  
 V_54       4.9967390e+00  
 V_55       4.9969144e+00  
 V_56       4.9971290e+00  
 V_57       4.9973826e+00  
 V_58       4.9976824e+00  
 V_59       4.9980368e+00  
 V_60       4.9984041e+00  
 V_61       4.9973990e+00  
 V_62       4.9972395e+00  
 V_63       4.9970981e+00  
 V_64       4.9969624e+00  
 V_65       4.9968535e+00  
 V_66       4.9967584e+00  
 V_67       4.9966868e+00  
 V_68       4.9966274e+00  
 V_69       4.9965377e+00  
 V_70       4.9964919e+00  
 V_71       4.9964896e+00  
 V_72       4.9965296e+00  
 V_73       4.9966073e+00  
 V_74       4.9967329e+00  
 V_75       4.9969034e+00  
 V_76       4.9971176e+00  
 V_77       4.9973651e+00  
 V_78       4.9976722e+00  
 V_79       4.9980230e+00  
 V_80       4.9983963e+00  
 V_81       4.9974217e+00  
 V_82       4.9972495e+00  
 V_83       4.9971008e+00  
 V_84       4.9969765e+00  
 V_85       4.9968635e+00  
 V_86       4.9967700e+00  
 V_87       4.9966944e+00  
 V_88       4.9966096e+00  
 V_89       4.9965239e+00  
 V_90       4.9964819e+00  
 V_91       4.9964815e+00  
 V_92       4.9965247e+00  
 V_93       4.9966107e+00  
 V_94       4.9967411e+00  
 V_95       4.9969066e+00  
 V_96       4.9971158e+00  
 V_97       4.9973718e+00  
 V_98       4.9976820e+00  
 V_99       4.9980260e+00  
